discussion/avoiding-mutable-aliases-to-hardware.tex

```diff
@@ -1,20 +1,22 @@
 % !TEX root = ../main.tex
 
-\section{Applying Ownership to Hardware}
-\label{sec:ownership_allied_to_hardware}
+\section{Avoiding Mutable Aliases to Hardware}
+\label{sec:avoiding_mutable_aliases_to_hardware}
 
 The problem that arises with aliasing to mutable data \cite{web:problem_with_shared_mutability}, also known as shared mutable state, might not be obvious to everyone.
```

Comment:
Hva med: `The problems that arises with aliasing to mutable data \cite{web:problem_with_shared_mutability}, also known as shared mutable state, are not always obvious.`